person.rb

```diff
@@ -0,0 +1,23 @@
+class Person
+  attr_reader :id
+  attr_accessor :name, :age
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+    @id = ''
+  end
+
+  private
+
+  def of_age?
+    p @age >= 18
+  end
+
+  public
+
+  def can_use_services?
+    true if is_of_age? || parent_permission
```

Comment:
- [ ] Good job so far 👍 This `parent_permission` here should be the instance variable. Kindly add the `@` symbol to it to fix that